doc/rtd/topics/debugging.rst

```diff
@@ -163,3 +163,63 @@ commandline:
 
 Inspect cloud-init.log for output of what operations were performed as a
 result.
+
+.. _proposed_sru_testing:
+
+Manually verifying cloud-init Stable Release Updates (SRU)
+==============================================================
+In order to retain stability on released Ubuntu series, cloud-init performs
+the following SRU procedure for `CloudinitUpdates`_.
+
+To manually test cloud-init **-proposed** versions
+before they are publicly released to **-updates** pockets:
+
+1. Launch a VM on your favorite platform, providing the cloud-config
+user-data `srutest.yaml`
+
+.. code:: bash
+
+    cat > srutest.yaml <<EOF
```

Comment:
People may be launching instances through their cloud's web UI, might it be better to just include the YAML and trust that CLI people will know how to write it to a file?

Reply:
+1 done.